app/helpers/application_helper.rb

```diff
@@ -1,4 +1,19 @@
-# frozen_string_literal: true
-
 module ApplicationHelper
+  def language_options
+    [['English', 'English'], ['Spanish', 'Spanish'],
+      ['Chinese', 'Chinese'], ['Vietnamese', 'Vietnamese'], ['France', 'France']]
```

Comment:
Since these are all the same, you shouldn't need to use the duplicates.
Also, should be `French`, not `France`.

['English', 'Spanish', 'Chinese', 'Vietnamese', 'French']